packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -70,10 +72,9 @@ exports.onPostBuild = (args, pluginOptions) => {
     globDirectory: rootDir,
     globPatterns,
     modifyUrlPrefix: {
-      rootDir: ``,
```

Comment:
I have no idea what the meaning of this was? `rootDir` is typically `"public"` maybe it was meant to be `[rootDir]` which also does not really make sense. Please review :)

Reply:
Yep, this was meant to be `[rootDir]`, my apologies :)